src/components/CorporateCampaign/CampaignPartner.vue

```diff
@@ -70,7 +70,11 @@ export default {
 		// make sure all partner content links open externally
 		if (this.$refs.partnerBodyCopy) {
 			const links = this.$refs.partnerBodyCopy.querySelectorAll('a');
-			links.forEach(link => { link.target = '_blank'; }); // eslint-disable-line no-param-reassign
```

Comment:
Haven't tested in IE, but I think Array.from would do the trick too
`Array.from(links).forEach(link => { link.target = '_blank'; }); // eslint-disable-line no-param-reassign)`

Reply:
Oh, nevermind. https://gomakethings.com/converting-a-nodelist-to-an-array-with-vanilla-javascript/. Unless maybe babel covers that by default.

Reply:
yep big ol 🤦  on my part. @emuvente helped me sort that out.